app/components/worktime-balance-chart/component.js

```diff
@@ -17,7 +17,7 @@ export default class WorktimeBalanceChart extends Component {
       datasets: [
         {
           data: this.args.worktimeBalances.map(({ balance }) =>
-            balance.asHours()
+            balance.asHours().toFixed(2)
```

Comment:
```suggestion
            Number.parseFloat(balance.asHours().toFixed(2))
```
This will remove all the insignificant trailing zeros. And then you won't have to change the test assertions.

Reply:
done